packages/cosmos_ui_components/lib/components/gradient_avatar.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:string_to_hex/string_to_hex.dart';
+
+class GradientAvatar extends StatelessWidget {
+  final String stringKey;
+
+  const GradientAvatar({Key? key, required this.stringKey}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final colors = generateColorsFromAddress(stringKey);
+    return CircleAvatar(
+      child: Container(
+        decoration: BoxDecoration(
+          shape: BoxShape.circle,
+          gradient: LinearGradient(colors: [colors[0], colors[1]]),
+        ),
+      ),
+    );
+  }
+
+  List<Color> generateColorsFromAddress(String address) {
```

Comment:
is this logic copied from web? we need to make sure we use the same logic as web version to stay consistent

Reply:
Not the same as web.

The web uses a package called [gradient avatar](https://www.npmjs.com/package/gradient-avatar) which actually does something like this inside it:
```
const hash = stringHash(str);
  const colors = hslTriad(hash % 360, 1, 0.5);
  const color1 = hslRgb(colors[0][0], colors[0][1], colors[0][2]);
  const color2 = hslRgb(colors[1][0], colors[1][1], colors[1][2]);
```

Reply:
I think I can make something similar to web using `HSLColor` class in dart.